modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -682,14 +692,20 @@ private CompletableFuture<?> createTablePartitionsLocally(
         int partitions = newAssignments.size();
 
         CompletableFuture<?>[] futures = new CompletableFuture<?>[partitions];
+
+        // TODO: https://issues.apache.org/jira/browse/IGNITE-19713 Process assignments and set partitions only for assigned partitions.
+        PartitionSet parts = new BitSetPartitionSet();
```

Comment:
maybe `new BitSetPartitionSet(partitions);`

Reply:
We don't need such a constructor. Right now we do iterate over all the partitions, but it's just incorrect (see ToDo)